### ISSUE TITLE
ofx plugin for davinci gives white screen

### ISSUE THREAD

**Issue by VBK7** [gyroflow/gyroflow-ofx#34]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you tried the latest build?

- [X] I have tried the latest build

### Gyroflow version

v1.5.0

### What operating system are you using?

macOS Ventura 13.3

### What GPU are you using?

AMD Radeon Pro 5500M 8 GB

### What happened?

When my resolution in my DaVinci Resolve project settings does not match the video and project resolution of my Gyroflow project, everything works as expected, but when the resolution of everything is the same, as soon as I load the .gyroflow file in the ofx plugin, it just gives me a white screen. I get the same results whether I apply the ofx plugin in the edit page or the color page and none of the settings within the plugin change anything. Very weird because I would think that issues would happen when the resolution doesn't match but in this case, it's the opposite.
<img width="1792" alt="gyroflow problem" src="https://user-images.githubusercontent.com/116237524/229309352-aa9b654c-720a-4b51-904e-d70a2bd6621a.png">

**Comment by VBK7** [comment]
[DaVinci-Resolve-logs-20230401-182932.zip](https://github.com/gyroflow/gyroflow-ofx/files/11130723/DaVinci-Resolve-logs-20230401-182932.zip)

**Comment by VBK7** [comment]
Also quick question, whats the difference between Project file, Project file (including gyro data) and, Project file (including processed gyro data)? The plugin seems to work exactly the same with each project file type, the only difference I see is file size. Also, is the plugin supposed to give 1:1 exactly the same results as the application if everything is done correctly because for me, it seems to always be a little bit different than if I were to stabilize the footage directly in the application. Is rolling shutter correction applied when using the plug-in?

**Comment by AdrianEddy** [comment]
> whats the difference between Project file

https://docs.gyroflow.xyz/app/advanced-usage/exporting#project-types
If you're using a video with embedded gyro data, they will all be the same. If you have external gyro data, you'll have to use "including gyro data". The third one is for external conversion scripts, doesn't matter for the plugin.

>  is the plugin supposed to give 1:1 exactly the same results

Yes, although there may be a bug -> #31 I didn't check that yet

> Is rolling shutter correction applied when using the plug-in?

Yes

**Comment by VBK7** [comment]
> > is the plugin supposed to give 1:1 exactly the same results
> 
> Yes, although there may be a bug -> #31 I didn't check that yet
> 
I did a test on an older version of the plugin and application where I processed a video through the application, and through the plugin in DaVinci Resolve, and they were pretty clearly not the same. The video file that I used for this test, I purposely turned the camera side to side very fast to see how the well the rolling shutter correction works. The stabilization looked pretty similar to the app but the rolling shutter correction seemed to be about half as much as through the app.

> > Is rolling shutter correction applied when using the plug-in?
> 
> Yes
Would be nice to see a slider and an on/off option for this in the future.
